Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/Spawnable/InMemorySpawnableAssetUtils.cpp

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+
+#include <AzCore/Asset/AssetManager.h>
+#include <AzCore/Asset/AssetManagerBus.h>
+#include <AzCore/Asset/AssetSerializer.h>
+#include <AzToolsFramework/Prefab/PrefabLoader.h>
+#include <AzToolsFramework/Prefab/PrefabSystemComponentInterface.h>
+#include <AzToolsFramework/Prefab/Spawnable/PrefabConverterStackProfileNames.h>
+#include <AzToolsFramework/Prefab/Spawnable/InMemorySpawnableAssetUtils.h>
+
+namespace AzToolsFramework::Prefab::PrefabConversionUtils
+{
+    InMemorySpawnableAssetProcessor::~InMemorySpawnableAssetProcessor()
+    {
+        Deactivate();
+    }
+
+    bool InMemorySpawnableAssetProcessor::Activate(AZStd::string_view stackProfile)
+    {
+        AZ_Assert(!IsActivated(),
+            "InMemorySpawnableAssetProcessor - Unable to activate an instance of InMemorySpawnableAssetProcessor as the instance is already active.");
+
+        m_prefabSystemComponentInterface = AZ::Interface<PrefabSystemComponentInterface>::Get();
+        AZ_Assert(m_prefabSystemComponentInterface, "InMemorySpawnableAssetProcessor - Could not retrieve instance of PrefabSystemComponentInterface");
+
+        m_loaderInterface = AZ::Interface<Prefab::PrefabLoaderInterface>::Get();
+        AZ_Assert(m_loaderInterface, "InMemorySpawnableAssetProcessor - Could not retrieve instance of PrefabLoaderInterface");
+
+        m_stackProfile = stackProfile;
+        return m_converter.LoadStackProfile(m_stackProfile);
+    }
+
+    void InMemorySpawnableAssetProcessor::Deactivate()
+    {
+        m_assetContainer.ClearAllInMemorySpawnableAssets();
+        m_stackProfile = "";
```

Comment:
This maybe a style thing but for clarity I would prefer `= {};`

Reply:
Is that actually equivalent for a string_view or would that be non-null terminated?